Electrino/Electrino/ENOJavaScriptApp.m

```diff
@@ -88,6 +126,18 @@ - (id)init
     return self;
 }
 
+// Create a new context for just evaluating the file
+// ISSUE: JSContext does not include -copyWithZone: method, so we have to manually copy the required methods.
+-(JSContext*)newContextForEvaluation
+{
+	JSContext* newContext = [[JSContext alloc] initWithVirtualMachine:self.jsVM];
+	newContext[@"require"] = self.jsContext[@"require"];
+	newContext[@"process"] = self.jsContext[@"process"];
+	newContext[@"console"] = self.jsContext[@"console"];
+	[newContext evaluateScript:@"var exports = {};"]; // Evaluated so the developer doesnt have to
```

Comment:
Sounds good! Just a node I originally used `exports` due to [this article](https://docs.npmjs.com/getting-started/creating-node-modules). 

I've done a bit more research on this, and it does seem a bit confusing about the whole `exports` vs `module.exports`. 

Regarding your suggested changes, how would setting `exports` also change `module.exports`? Unless I'm missing something, we are only exposing `module`, unless you propose that we leave both, and we check which variable is populated...